### ISSUE TITLE
Non map size generation [custom total size in blocks feature request]

### ISSUE THREAD

**Issue by danivenk** [rebane2001/mapartcraft#287]
So I've been using the amazing Mapartcraft to make maparts for a while now on my server. However, currently, we're working on a perimeter to which I would like to add a kind of mapart to such that it would show up nicely on the world map. I have however one problem, the perimeter is not the size of a map nor is it centered around one.
As such I would like to request a feature for the generation of such "mapart" with custom sizes in actual blocks instead of maps.
Yes, I have tried to look at alternatives but either the dittering algorithms were not as extensive or they wouldn't be able to export the result to a schematic.
If this would be possible to be added that would really be appreciated.

**Comment by PapaLoong** [comment]
Yes, I also need this feature, although 64x64 bit does not work well, but I wish I could implement this feature

**Comment by rebane2001** [comment]
fyi it's possible to use fractional values in the classic version

**Comment by danivenk** [comment]
It would be nice to have this too in the current version of the tool too. Maybe even that it would calculate the fraction if the user fills in. Altho ideally I would like to be able to input an actual block count, but if that is too much work reimplementing the fraction inputs from the classic version is also fine by me.

Please do whatever you feel is the right way and what you have the time for to do, after all this is just a feature request and there are technically ways to circumvent it by hacking the input image file to get what you want (by padding it to the size of a multiple of 128).

**Comment by rebane2001** [comment]
yeah it definitely should have it, but i'm not all that active with dev right now so i just wanted to mention it as an alternative for now
